### PR TITLE
Unit tests for model_loader.py

### PR DESCRIPTION
**Pull request by piyushghai**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by piyushghai** [comment]
Waiting for https://github.com/awslabs/mxnet-model-server/pull/456 to be merged, because it directly conflicts with the changes I have made to model_loader file. 
Will update this PR after #456 gets merged.

File: mms/tests/unit_tests/test_model_loader.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+    with pytest.raises(Exception) as error:
+        ModelLoader.load(model_dir_path, handler_file)
+
+    assert error.value.args[0] == 'parameterFile not found: {}.'.format(manifest_valid_data['model']['parametersFile'])
+
+
+@pytest.mark.usefixture('create_empty_manifest_file')
+def test_symbol_file_defined_in_manifest():
+    with open(model_dir_path + '/' + MANIFEST_FILENAME, 'w') as f:
```
Comment (vrakesh) [inline mms/tests/unit_tests/test_model_loader.py]: use string join function

File: mms/tests/unit_tests/test_validate_messages.py [hunk 2]
```diff
@@ -15,25 +15,35 @@
...
-    assert error.value.get_message() == 'Load command missing \"model-name\" key'
+    assert error.value.get_message() == 'Load command missing \"modelName\" key'
+
+
+def test_validate_load_messages_missing_handler():
+    invalid_object = {'command': 'load', 'modelPath': 'some-model-path', 'modelName': 'some model name'}
+    with pytest.raises(MMSError) as error:
+        ModelWorkerMessageValidators.validate_load_message(invalid_object)
+
+    assert error.value.get_code() == ModelServerErrorCodes.INVALID_LOAD_MESSAGE, 'error codes don\'t match'
```
Comment (vrakesh) [inline mms/tests/unit_tests/test_validate_messages.py]: Use double quotes for outer string

"error codes don't"

File: mms/utils/validators/validate_messages.py [hunk 3]
```diff
@@ -35,10 +35,13 @@ def validate_load_message(msg):
         :return:
         """
         if u'modelPath' not in msg:
```
Comment (vrakesh) [inline mms/utils/validators/validate_messages.py]: why is unicode still being mentioned, we have imported future right?

File: mms/model_loader.py [hunk 4]
```diff
@@ -58,13 +58,13 @@ def load(model_dir, handler):
                 parameter_file = model.get('parametersFile')
                 if not parameter_file:
                     raise Exception("parameterFile not defined in MANIFEST.json.")
-                if os.path.isfile(parameter_file):
+                if not os.path.isfile(parameter_file):
```
[the remark this answers is not in the document]
Comment (piyushghai) [reply mms/model_loader.py]: Sure. Will remove that as well.

[the remark this answers is not in the document]
Comment (piyushghai) [reply mms/model_loader.py]: Sure. Will remove those as well.

**Comment by piyushghai** [comment]
Messed something here. Closing this PR.

**Pull request closed without merging**